openedx_webhooks/tasks/pr_tracking.py

```diff
@@ -386,6 +387,7 @@ def fix(self) -> None:
         if fix_comment:
             self._fix_bot_comment(comment_kwargs)
         self._add_bot_comments()
+        self.actions.update_commit_status_for_cla(pull_request=self.pr)
```

Comment:
This bit, the `self.actions...` part, seems to be how we're performing dry-runs?